docs/deployment/settings.md

```diff
@@ -296,9 +296,16 @@ kyuubi.frontend.protocols|THRIFT_BINARY|A comma separated list for all frontend
 kyuubi.frontend.proxy.http.client.ip.header|X-Real-IP|The http header to record the real client ip address. If your server is behind a load balancer or other proxy, the server will see this load balancer or proxy IP address as the client IP address, to get around this common issue, most load balancers or proxies offer the ability to record the real remote IP address in an HTTP header that will be added to the request for other devices to use. Note that, because the header value can be specified to any ip address, so it will not be used for authentication.|string|1.6.0
 kyuubi.frontend.rest.bind.host|&lt;undefined&gt;|Hostname or IP of the machine on which to run the REST frontend service.|string|1.4.0
 kyuubi.frontend.rest.bind.port|10099|Port of the machine on which to run the REST frontend service.|int|1.4.0
+kyuubi.frontend.ssl.exclude.protocols|SSLv2,SSLv3|SSL Versions to disable.|seq|1.7.0
```

Comment:
kyuubi.frontend(.binary).ssl.disallowed(disabled)Protocols, the doc part can be either removed as it says nothing more than the config name, or we can elaborate it a bit more.